source/TLS.rst

```diff
@@ -0,0 +1,42 @@
+TLS
```

Comment:
this doc should describe to the user that he needs to configure internal and external TLS certificates.
explain that it is possible to disable TLS with `allowUnsafeConfigurations` and provide steps on how to generate certificates manually or send user installing cert-manager.

Reply:
Sure. Please provide more information and examples on how someone would configure internal and external TLS certificates, how to disable TLS, and how to generate certificates. 

https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/

How is what we want different than the options in the link? What are the specifically Percona-based methods?

Reply:
we can have a call if you need more information.
"managing-tls-in-a-cluster" link is **completely** about another thing.